src/JarvisDesk.cpp

```diff
@@ -203,33 +220,42 @@ class JarvisDesk_impl {
     return false;
   }
 
-  void unlatch()
-  {
+  void unlatch() {
     unlatch_pin(HS0);
     unlatch_pin(HS1);
     unlatch_pin(HS2);
     unlatch_pin(HS3);
   }
 
-  void latch(unsigned preset)
-  {
-    if (!preset || preset > 4) return;
+  void latch(unsigned preset) {
+    if (!preset || preset > 4)
+      return;
 
     // pinout pattern has preset+2
     preset += 2;
 
     unlatch();
-    if (preset & 1) latch_pin(HS0);
-    if (preset & 2) latch_pin(HS1);
-    if (preset & 4) latch_pin(HS2);
-    if (preset & 8) latch_pin(HS3);
+    if (preset & 1)
+      latch_pin(HS0);
+    if (preset & 2)
+      latch_pin(HS1);
+    if (preset & 4)
+      latch_pin(HS2);
+    if (preset & 8)
+      latch_pin(HS3);
   }
+#if defined(JCB35N2PA32V2)
```

Comment:
Oh, is this the 0x01 0x01 protocol I saw before?  Nice.

Reply:
yes, makes it a bit difficult with how simplistic it is

Reply:
I have tricked my controller into using this mode accidentally. I don't know how. I wonder what your handset does to force it always into this mode. It woulc be nice to see a logic analyzer capture of the signals on startup from your controller.

Reply:
I will add it to the list of things to capture. I would rather use your packet mode (if that is even possible) anyways.